src/apps/administration/index.md

```diff
@@ -0,0 +1,41 @@
+
+## Build
+
+The [Administration module](https://developer.shopware.com/docs/concepts/framework/architecture/administration-concept) contains infrastructure and assets defined and used in the Administration user interface.
+
+You can modify the appearance and extend capabilities of the Administration by customizing its components such as [template files](https://developer.shopware.com/docs/guides/plugins/plugins/administration/writing-templates), [mixins](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-mixins), [fields](https://developer.shopware.com/docs/guides/plugins/plugins/administration/using-custom-fields), [filters](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-filter), [modules](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-module), [styling](https://developer.shopware.com/docs/guides/plugins/plugins/administration/add-custom-styles), etc.
+
+This can be done via [apps](https://developer.shopware.com/docs/guides/plugins/apps/administration) or [plugins](https://developer.shopware.com/docs/guides/plugins/plugins/administration).
+
+### Rules
+
+You can create [custom rules](https://developer.shopware.com/docs/guides/plugins/plugins/framework/rule/add-custom-rules) to handle almost anything ecommerce-related for the customer, product or cart. It gives you control over what you want to happen if a specific condition is fulfilled.
+
+### Flow Builder
+
+[Flow Builder](https://developer.shopware.com/docs/guides/plugins/plugins/framework/flow/add-flow-builder-action#add-custom-action-in-administration) lets you build applications to automate a business process.
+
+### CMS
+
+Once the Administration component is built, the representation is handled by [CMS](https://developer.shopware.com/docs/guides/plugins/plugins/content/cms)
+
+### Admin API
+
+Building the Administration component also involves creating necessary API interfaces and services. 
+
+To provide CRUD operations for every entity within Shopware, the [Admin API](https://developer.shopware.com/docs/concepts/api/admin-api) offers a full range of commerce options making it possible for an ecommerce business to handle backend data.
+
+Coding standards, like for many large projects, Shopware Open Source also has [coding standards](https://developer.shopware.com/docs/resources/guidelines/code). Use these standards when you contribute to the codebase or create extensions.
+
+## Compile
+
+Administration loads configurations from the database and compiles them. To allow compiling without a database, refer to this [guide](https://developer.shopware.com/docs/guides/hosting/installation-updates/deployments/build-w-o-db#compiling-the-administration-without-database)
```

Comment:
The compilation is more of a step towards a working Administration.

Reply:
Removed!